charts/fluentd-elasticsearch/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: fluentd-elasticsearch
-version: 5.0.1
+version: 5.0.2
```

Comment:
as this a new feature version should be raised to 5.1.0

Reply:
Done